src/util/Util.ts

```diff
@@ -286,3 +288,85 @@ export function timedPromise<T>(wrapped: Promise<T>, timeout: number, err?: Erro
 
     return deferred.promise;
 }
+
+/**
+ * Finds a larger same-version group of data members from a collection of
+ * members and, if `localMember` is from that group, return that.
+ * Otherwise return a random member from the group. If the same-version
+ * groups have the same size, return a member from the newer group
+ * (preferably the local one).
```

Comment:
local member does not make sense for a client, so we might update the documentation